bundle/manifests/windows-prometheus-k8s-rules_monitoring.coreos.com_v1_prometheusrule.yaml

```diff
@@ -45,3 +45,9 @@ spec:
     - expr: |
         label_replace(windows_container_memory_usage_private_working_set_bytes * on(container_id) group_left(namespace, pod, container) kube_pod_container_info{container_id!=""},"container","","","")
       record: container_memory_working_set_bytes
+    - expr: |
```

Comment:
Please test this through https://github.com/openshift/windows-machine-config-operator/blob/8a4f95f905527ece51fa9fb2580764a7ff6645a4/test/e2e/metrics_test.go#L275-L276